packages/origin-backend/src/pods/mail/mail.module.ts

```diff
@@ -0,0 +1,44 @@
+import { Module } from '@nestjs/common';
+import { HandlebarsAdapter, MailerModule } from '@nestjs-modules/mailer';
+import mandrillTransport from 'nodemailer-mandrill-transport';
+import { ConfigService } from '@nestjs/config';
+import { MailService } from './mail.service';
+
+@Module({
+    imports: [
+        MailerModule.forRootAsync({
+            useFactory: async (configService: ConfigService) => {
+                return {
+                    transport: mandrillTransport({
+                        auth: {
+                            apiKey: configService.get<string>('MANDRILL_API_KEY')
+                        },
+                        // eslint-disable-next-line @typescript-eslint/camelcase
+                        from_email: configService.get<string>('EMAIL_FROM'),
+                        // eslint-disable-next-line @typescript-eslint/camelcase
+                        from_name: 'Energy Web Origin',
+                        headers: {
+                            'Reply-To': configService.get<string>('EMAIL_REPLY_TO')
+                        },
+                        merge: true,
+                        tags: ['origin', 'no-reply']
+                    }),
+                    defaults: {
+                        from: `"Energy Web Origin" <${configService.get<string>('EMAIL_FROM')}>`
+                    },
+                    template: {
+                        dir: `${__dirname}/templates`,
+                        adapter: new HandlebarsAdapter(),
+                        options: {
+                            strict: true
+                        }
+                    }
```

Comment:
Are these templates used in any way?

Reply:
They aren't. but can be, if we decide to use them.